libraries/AP_OSD/AP_OSD.cpp

```diff
@@ -114,10 +114,26 @@ const AP_Param::GroupInfo AP_OSD::var_info[] = {
 
     // @Param: _W_RSSI
     // @DisplayName: RSSI warn level (in %)
-    // @Description: Set level at which RSSI item will flash
-    // @Range: 0 99
+    // @Description: Set level at which RSSI item will flash (in % or dBm if applicable)
```

Comment:
```suggestion
    // @Description: Set level at which RSSI item will flash (in positive % or negative dBm values as applicable). 30% or -100dBm are defaults.
```

Reply:
Another great idea! Will update like this.